src/index.js

```diff
@@ -1,13 +1,46 @@
-import _ from 'lodash';
 import './style.css';
+import modal from './modules/displayGameComments.js';
+import Comments from './modules/comments.js';
+import newComment from './modules/commentsApi.js';
+import getComments from './modules/getComments.js';
+import options from './modules/freegames.js';
+import likeSend from './modules/sendlikes.js';
 
-function component() {
-  const element = document.createElement('div');
+// window.addEventListener('load', () => {
+options();
+const body = document.querySelector('#body');
+body.addEventListener('click', (e) => {
+  if (e.target.textContent === 'Comments' && e.target.className !== 'like-button') {
+    const games = JSON.parse(localStorage.getItem('allGames'));
+    const id = e.target.className.split(' ')[1];
+    games.forEach((game) => {
+      if (id === game.id) {
+        modal(id);
+        getComments(id);
+      }
+    });
+    const closeBtn = document.querySelector('#close');
+    const newPop = document.querySelector('.new-pop');
+    closeBtn.addEventListener('click', () => {
+      newPop.style.display = 'none';
+    });
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-
-  return element;
-}
-
-document.body.appendChild(component());
\ No newline at end of file
+    const submit = document.querySelector('#submit');
+    submit.addEventListener('click', (e) => {
+      e.preventDefault();
+      const newId = e.target.className;
+      const name = document.querySelector('#name');
+      const comment = document.querySelector('#comment');
+      const date = new Date();
+      const day = date.getDate();
+      const month = date.getMonth() + 1;
+      const year = date.getFullYear();
+      const currentDate = ` ${month}/ ${day}/ ${year}`;
+      Comments.addComment(name.value, comment.value, newId, currentDate);
+      newComment(name.value, comment.value, newId);
+    });
+  } else if (e.target.textContent !== 'Comments' && e.target.className === 'like-button') {
+    const { id } = e.target;
+    likeSend(id);
```

Comment:
- [ ] Sometimes the likes button on your game cards don't work 🐛. When you try to like a video, sometimes it works and sometimes it doesn't. Please try to fix it 👍 
![ezgif com-gif-maker](https://user-images.githubusercontent.com/46839548/197206407-b4d349dd-9202-47b7-a14c-e7642a3530cb.gif)

Reply:
@codepantha hey mate, I have checked the code again in different browsers, and it is working, I don't know why it is not working at your end, can you please tell me how can I mock the error on my local machine?

Reply:
@shakerAbuDrais, sorry for the late response. I see your project has been approved. Well, just simply by clicking on all the 'like' buttons you have, you can see that some of them don't work. I believe @zainsadaqat has thrown more light on the issue if you would kindly look at it. Thanks 🙏